lib/_stream_writable.js

```diff
@@ -285,9 +285,15 @@ Writable.prototype.write = function (chunk, encoding, cb) {
   var state = this._writableState;
   var ret = false;
 
-  var isBuf = !state.objectMode && _isUint8Array(chunk);
+  // var isBuf = !state.objectMode && _isUint8Array(chunk);
 
-  if (isBuf && !Buffer.isBuffer(chunk)) {
```

Comment:
Similarly, I see no reason why `!Buffer.isBuffer(chunk)` check should be removed for a UInt8Array?